test/02.basic-querying.test.js

```diff
@@ -4,7 +4,7 @@ var db, User, Customer, AccessToken, Post, PostWithId, Category, SubCategory;
 
 /*eslint no-console: "off"*/
 /*global getSchema should*/
-describe('basic-querying', function () {
+xdescribe('basic-querying', function () {
```

Comment:
@pulkitsinghal since I should not change existing tests, I copied the basic-querying tests and removed all timeouts (since my last PR regarding refresh: true this is possible)
- 03.basic-querying-no-timeout.test.js
The tests are the same as the other tests but are way faster ... if you like, you can reenable them or remove them completely (my suggestion)